### ISSUE TITLE
Reimbursement for b7ijeetkdmfg2vzgmsbwlent3d6uqt7o6ysgnue54t6bks5pcym7f6yd.onion:9999

### ISSUE THREAD

**Issue by joakimedberg** [bisq-network/support#1571]
![rebisq](https://github.com/bisq-network/support/assets/34625992/dff9c73a-e86b-4690-b496-619d11f5f7b2)

Reimbursement for trade ei0qfcu

I am BTC seller as maker.
Trade amount and asset: 0.01 BTC for 394 EUR
Deposit amount: 0.0015 BTC

Maker:  a08b2fe201233c96679a8838dec7042e36fc419fcc5a7464e41e193576b78ee5
Taker: c239c6432c41b4a8ea5beec7f3fbbf967a0ef366b2d7cfd8c3837983ec47995d
Deposit: ea5a80ef345261bbf696d5d92a671b8744f72972facc8226f75fb48371960303
Delayed payout: 84ed8f87d88f0af8d6f98b7a6b6bc650b87d9438119dae325aa1f19c87464aaf

Reimbursement amount: 0.01 BTC / 0.00003794 BSQ = 263.57 BSQ
Reason for reimbursement: Mediator reversed payouts in a canceled trade. "Buyer" not responding.

@refund-agent2 @pazza83

**Comment by pazza83** [comment]
Hi @joakimedberg 

Thanks for posting this issue.

I can confirm that the proposal I made was issued incorrectly. The trade should have been cancelled and my proposal was to cancel the trade but to give the buyer the trade amount.

Both yourself and the buyer accepted the incorrect proposal and as a result the buyer was paid the trade amount rather than yourself.

I agree you should be refunded the 0.01 BTC or BSQ equivalent as the mistake was largely down to the incorrect mediation proposal I made.

-----------------
Copy of the mediation proposal

Ticket closed on 25 Jan 2024 19:04:24
Mediator node address: d7m3j3u4jo2yuymgvxisklpitd3n5xbsnnpyz2mjh6bl6gmj5rjdxead.onion:9999

Summary:
Trade ID: ei0qfcu
Currency: EUR
Reason for dispute: Bank
Payout suggestion: BTC buyer gets trade amount payout
Trade amount: 0.01 BTC
Payout amount for BTC buyer: 0.0115 BTC (BTC buyer receives trade amount + his deposit)
Payout amount for BTC seller: 0.0015 BTC (BTC seller receives his deposit)

Summary notes:
BTC buyer was unable to send payment to BTC seller.
BTC seller confirmed no payment received

Proposing trade to be cancelled

-----BEGIN SIGNATURE-----
302d02150084f5539a5e72bc9344e67a0774433493aece8512021476356c373d0206355eecad45998bc295bbe34320
-----END SIGNATURE-----

Next steps:
Go to Open trades to Accept the mediation proposal and wait for your peer's acceptance, if necessary.
Click Reject if you disagree and explain your reasons on the mediation ticket.
If the trade has not been paid out in the next 48h, please inform your mediator on this ticket.
Keep in mind that the arbitrator can only make the payout of the trade amount + 1 security deposit. The other security deposit will be left unpaid.

**Comment by joakimedberg** [comment]
Proposal transaction ID: b28c280aa991c3bb127e899359c9f352981a925ca4eae67c07f118e1ff24c1a5

**Comment by pazza83** [comment]
Thanks for positing, I agree that you should be reimbursed. 

The vote will take place between 2nd-5th March. 

The result of the vote will be published 9th March

**Comment by MwithM** [comment]
I  have doubts about reimbursing the full amount.

Buyer and seller need to read and accept the mediation proposal, it's not enforced, and the mistake can be corrected with a manual payout. So, although it's clear that mediator did not submit the correct proposal, in the end it's a user error, who accepted a wrong proposal.

At least, we should determine what could be done for the next time, so this does not happen again.

If this reimbursement proposal is accepted, it should not set a precedent for other trades as this create a vulnerability for the DAO.

**Comment by pazza83** [comment]
I agree it should not set a precedent.

I think my error was made worse in that the text of my proposal read that the trade would be cancelled, yet the payout amounts said otherwise.

I agree the @joakimedberg also made an error by accepting my incorrect proposal.

> At least, we should determine what could be done for the next time, so this does not happen again.

Agreed, for this trade I am have been trying contact the BTC buyer that got the 0.01 BTC in error. They have read my replies but not responded. 

The error was a result of 'human error'. I have issued about 3,000 mediation proposals and this was the first one that I have issued the wrong payout proposal for.  

From a personal reflection I think I was a little tired when I made the proposal. Since the error I have been avoiding mediating when tired and also taking extra case to double check things. I do think human errors cannot be totally eliminated though. 

Going forward maybe an agreed limit of 0.01 BTC worth can be compensated this way. That way it limits the DAOs exposure. 

I also considered reimbursing the user myself, something I still might do if the reimbursement is not accepted by the DAO, however I did not want to set a precedent for a mediator reimbursing for their own mistakes either.
